src/api/common/schemas/get-paths.json

```diff
@@ -6,7 +6,23 @@
     "type": "object",
     "properties": {
       "source": {"$ref": "adjustment"},
-      "destination": {"$ref": "adjustment"},
+      "destination": {
+        "type": "object",
```

Comment:
Taking a step back, David said in the chat that:

> It's not a special case. The destination is the issuer. Which end of a trust line you consider the "issuer" is arbitrary. You are an issuer for all your trust lines.

With this interpretation we could say that it's not actually wrong or even really a special case, so we could leave it as is.

Basically we are adding a syntactic sugar that allows the user to just not specify the counterparty when they don't care about the counterparty, which I think is desirable for user-generated requests.

The question here is whether we should apply the syntactic sugar even for non-user-generated responses. I think it's confusing either way; but I guess it's better to do it as you suggest.